webknossos/webknossos/proofreading/agglomerate_graph_pb2.pyi

```diff
@@ -0,0 +1,155 @@
+from collections.abc import Iterable as _Iterable
```

Comment:
could the generated files be in a folder named “generated” or similar, to mark this for other developers?